### DISCUSSION TITLE
Welcome to multiscript discussions!

### DISCUSSION THREAD

**Discussion by jamesbcd** [multiscript/multiscript#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dcpurton** [comment]
@jamesbcd I reckon I could make use of this with an appropriate LaTeX output template.

I'm currently working on a release worthy version of my LaTeX macros for typesetting Bible quotations which once done, will appear in the various standard TeX distributions. It will make it easy to come up with a template that will produce a nice looking PDF (though font and language handling might need a bit of thought).

Once my package is released, I can help you get something working.
